tools/perf/brave_page_sets/brave_loading_desktop_pages.py

```diff
@@ -33,26 +33,22 @@ def _StartBrowser(self, page):
     time.sleep(10)
 
 
-class BraveLoadingDesktopStorySet(story.StorySet):
-  """ Brave version of LoadingDesktopStorySet.
+class BaseBraveLoadingDesktopStorySet(story.StorySet):
```

Comment:
BTW, the original chromium benchmark `loading.desktop` was discounted sometime ago.
One day we could also drop `loading.desktop.brave` in favor of `system_health.*`

Reply:
I have created another [PR](https://github.com/brave/brave-core/pull/25193) with `system_health...`, So probably I can close this PR.